README.md

```diff
@@ -29,8 +73,24 @@ The peer id will be printed to stdout.
 
 If you want to enable gRPC/REST, add the following entries to your config.yml:
 
-    listenGrpcMultiaddr: <multiaddr> 
-    listenRESTMultiaddr: <multiaddr>
+    sudo nano .config/config.yml
+
+edit these lines below
+
+    listenGrpcMultiaddr: /ip4/127.0.0.1/tcp/8337
+    listenRESTMultiaddr: /ip4/127.0.0.1/tcp/8338
+
+Save and exit
+
+Ensure that port 8337 among other neeeded ports are enabled via firewal.
+
+    sudo ufw enable
+    sudo ufw allow 22
+    sudo ufw allow 8336
+    sudo ufw allow 8337
+    sudo ufw allow 8338
```

Comment:
Why would you open port 8337 and 8338? In line 80 and 81 you restrict them to localhost (127.0.0.1). Additionally, from the README: "Please note: this interface, while read-only, is unauthenticated and not rate- limited. It is recommended that you only enable if you are properly controlling access via firewall or only query via localhost."